plugins/base/src/main/kotlin/signatures/KotlinSignatureProvider.kt

```diff
@@ -117,6 +120,19 @@ class KotlinSignatureProvider(ctcc: CommentsToContentConverter, logger: DokkaLog
                 } ?: regularSignature(c, sourceSetData)
         }
 
+    private fun <T : Documentable> PageContentBuilder.DocumentableContentBuilder.defaultValueAssign(
+        d: WithExtraProperties<T>,
+        sourceSet: DokkaSourceSet
+    ) {
+        // a default value of parameter can be got from expect source set
+        d.extra[DefaultValue]?.expression?.let {
+            it[sourceSet] ?: if (d is DParameter) it[d.expectPresentInSet] else null
+        }
+            ?.let { expr ->
+                operator(" = ")
+                highlightValue(expr)
+            }
```

Comment:
Gap looks strange, just a suggestion
```suggestion
        d.extra[DefaultValue]?.expression
            ?.let { it[sourceSet] ?: if (d is DParameter) it[d.expectPresentInSet] else null }
            ?.let { expr ->
                operator(" = ")
                highlightValue(expr)
            }
```